prog/vnet/update_load_balancer.rb

```diff
@@ -18,7 +18,10 @@ def load_balancer
   end
 
   label def remove_load_balancer
-    vm.vm_host.sshable.cmd("sudo ip netns exec #{vm.inhost_name} nft --file -", stdin: generate_nat_rules(vm.ephemeral_net4.to_s, vm.nics.first.private_ipv4.network.to_s))
+    if vm
```

Comment:
Nit: 
>  Load Balancer archival is fixed in case the VM is already archived 

Maybe like:

> Fix crash when VM is concurrently deleted

This is only to make it shorter and a bit clearer.

More generally: is it right for update load balancer to be orthogonal/not synchronized with the VM flow of control?

Granted, it's better to commit this given the status quo than not, but perhaps this prog ought to be triggered via VM semaphore unless it's necessary to the VM doing, possibly, anything while doing this.

Reply:
I have initially designed it to be all under a single entity LoadBalancer, moving forward, it's possible that the VM sourced flow is better but my dilemma is, whenever we perform a vm specific load balancer update, we need to update the other load balancer nodes as well. I guess, for now, I will opt into watch and see.